### PR TITLE
Controller-Tools and Go version Bump

### PR DESCRIPTION
**Pull request by kensipe**
Signed-off-by: Ken Sipe <kensipe@gmail.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -31,6 +31,6 @@ require (
 	k8s.io/client-go v0.19.2
 	k8s.io/code-generator v0.19.2
 	sigs.k8s.io/controller-runtime v0.6.3
-	sigs.k8s.io/controller-tools v0.3.0
+	sigs.k8s.io/controller-tools v0.4.0
```
Comment (estroz) [inline go.mod:34]: FYI there's a v0.4.1 patch
```suggestion
	sigs.k8s.io/controller-tools v0.4.1
```

File: go.mod [hunk 2]
```diff
@@ -31,6 +31,6 @@ require (
 	k8s.io/client-go v0.19.2
 	k8s.io/code-generator v0.19.2
 	sigs.k8s.io/controller-runtime v0.6.3
```
Comment (alenkacz) [inline go.mod:33]: there's also 0.6.4 and 0.7.0 for this. I think at least the 0.6.4 bump should be easy

File: go.mod (hunk 1 above)
Comment (alenkacz) [reply go.mod:34]: 👍

**Review by alenkacz — APPROVED** [review]
LGTM, maybe we can do two more minor bumps though

**Comment by kensipe** [comment]
thanks @estroz

**Pull request merged**